client/src/components/UserShowPage.js

```diff
@@ -64,13 +64,15 @@ const UserShowPage = (props) => {
             console.error(`error in fetch: ${error}`)
           }
         }
+        console.log(user)
+        console.log(currentUser)
 
         useState(() => {
           getUser()
         }, [])
 
     let dropzoneComponent = ""
-    if(currentUser){
+    if(currentUser?.id === user?.id){
```

Comment:
should not need conditional operator `?` for `user.id` as that is explicitly defined in state with the users properties